tests/parsers/test_dos.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=unused-argument
+"""Tests for the `DosParser`."""
+from __future__ import absolute_import
+import pytest
+from aiida.orm import FolderData
+from aiida.common import AttributeDict
+
+
+@pytest.fixture
+def dos_inputs():
+    # The DosParser doesn't really need to access the parent folder, but we'd like to make the inputs as realistic
+    # as possible, so we create an empty FolderData and attach it as an input to the current CalcJobNode.
+    inputs = {
+        'parent_folder': FolderData().store(),
```

Comment:
Just for future reference, there is not need to call `store` manually here

Reply:
Ah, thanks.